### ISSUE TITLE
ETA for stable 2.0 

### ISSUE THREAD

**Issue by ericnewton76** [commandlineparser/commandline#77]
<a href="https://github.com/OsirisTerje"><img src="https://avatars0.githubusercontent.com/u/203432?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [OsirisTerje](https://github.com/OsirisTerje)**
_Monday Jun 27, 2016 at 08:07 GMT_
_Originally opened as https://github.com/gsscoder/commandline/issues/328_

----

Are there any chance of getting a stable version of the 2.0 out?

**Comment by ericnewton76** [comment]
<a href="https://github.com/nemec"><img src="https://avatars3.githubusercontent.com/u/136378?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nemec](https://github.com/nemec)**
_Wednesday Jun 29, 2016 at 19:16 GMT_

----

Only @gsscoder has access to do that and he currently has other commitments to attend to. If you're concerned about crashing the latest master is stable, but unfortunately it's not available via NuGet.

**Comment by ericnewton76** [comment]
<a href="https://github.com/OsirisTerje"><img src="https://avatars0.githubusercontent.com/u/203432?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [OsirisTerje](https://github.com/OsirisTerje)**
_Wednesday Jun 29, 2016 at 19:42 GMT_

----

It seems stable enough to me, I have been using it for more than 6 months now (the latest beta on nuget) , and it works just fine.  I am mostly concerned that it has a beta version number, and when it is marked as a dependency in another package that package also has to have a beta version number.  So it is mostly annoying, can work around it by not declaring it, but not so nice.  But guess we have to wait on @gsscoder .

**Comment by ericnewton76** [comment]
<a href="https://github.com/Mizipzor"><img src="https://avatars1.githubusercontent.com/u/139879?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Mizipzor](https://github.com/Mizipzor)**
_Saturday Aug 06, 2016 at 11:27 GMT_

----

@nemec do you know the current status on the stable? I have some time to spend on this project right now and I'm really keen to fix the remaining bugs that we would require for stable. But I dont know the roadmap. Even if @gsscoder is busy surely he can just put in a new nuget version if the source is in a good place.

**Comment by ericnewton76** [comment]
<a href="https://github.com/nemec"><img src="https://avatars3.githubusercontent.com/u/136378?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nemec](https://github.com/nemec)**
_Sunday Aug 07, 2016 at 06:39 GMT_

----

You know, I'm not really sure. There is a 2.0 stable milestone, but comments within one of the issues indicate that it's postponed until post-2.0 release. TBH I'm happy to call it stable.

**Comment by ericnewton76** [comment]
<a href="https://github.com/zidad"><img src="https://avatars1.githubusercontent.com/u/303034?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [zidad](https://github.com/zidad)**
_Saturday Aug 13, 2016 at 12:23 GMT_

----

I'm running into the same issue as @OsirisTerje, I'd really like a stable release number to be able to use this project in other (non-beta) packages... @gsscoder any chance?

**Comment by ericnewton76** [comment]
<a href="https://github.com/igloo15"><img src="https://avatars3.githubusercontent.com/u/2975460?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [igloo15](https://github.com/igloo15)**
_Thursday Aug 25, 2016 at 03:22 GMT_

----

@gsscoder hasn't been active in a long time should someone maybe create a new nuget package of this?

**Comment by ericnewton76** [comment]
<a href="https://github.com/zidad"><img src="https://avatars1.githubusercontent.com/u/303034?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [zidad](https://github.com/zidad)**
_Monday Sep 12, 2016 at 16:56 GMT_

----

@igloo15 @OsirisTerje Funny! I tried to make an unofficial released package called CommandLineParser.Unofficial, but when I tried to push it to nuget I found out someone already had exactly the same idea:  https://www.nuget.org/packages/CommandLineParser.Unofficial

**Comment by ericnewton76** [comment]
<a href="https://github.com/nemec"><img src="https://avatars3.githubusercontent.com/u/136378?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nemec](https://github.com/nemec)**
_Saturday Nov 05, 2016 at 08:40 GMT_

----

I finally got access to the official repo and pushed a new prerelease package based on the latest master. #361 seems to be the only outstanding bug that should be resolved before we can release a stable 2.0.

https://www.nuget.org/packages/CommandLineParser/2.1.1-beta

**Comment by ericnewton76** [comment]
<a href="https://github.com/MgSam"><img src="https://avatars0.githubusercontent.com/u/1965570?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [MgSam](https://github.com/MgSam)**
_Tuesday Feb 21, 2017 at 15:21 GMT_

----

Any update on when 2.0 will be released? The NuGet publish date on 1.9.71 of  Feb 27, 2013 probably makes a lot of people think this project is dead.

Also- could someone update the CodePlex site so that it says at the top in large print that development has moved to Github? I had no idea this repo existed without doing a lot of digging- CodePlex is still one of the top hits on Google.

**Comment by ericnewton76** [comment]
<a href="https://github.com/hugener"><img src="https://avatars2.githubusercontent.com/u/5023998?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [hugener](https://github.com/hugener)**
_Monday Mar 13, 2017 at 18:25 GMT_

----

I posted a comment on what appears to be the last remaining bug: https://github.com/gsscoder/commandline/issues/361
It seems to be clear what's wrong, but I'm not sure if there is an easy fix.

**Comment by ericnewton76** [comment]
<a href="https://github.com/tesharp"><img src="https://avatars0.githubusercontent.com/u/5462839?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [tesharp](https://github.com/tesharp)**
_Thursday Jun 29, 2017 at 09:03 GMT_

----

Any chance to create a release package soon? Its been a year and looking at NuGet it seems a lot of people have just resolved to using the unofficial package by now.. Minor bugs can be fixed later as well. If the goal is to have a bug free release then I guess we have to wait another year.

**Comment by kenny-evitt** [comment]
@ericnewton76 @nemec This seems like it can be closed.
